components/buttongroup/metadata/buttongroup.yml

```diff
@@ -16,8 +18,27 @@ examples:
         </button>
       </div>
 
-  - id: buttongroup
+  - id: buttongroup-horizontal-small
+    name: Horizontal - Small
+    description: |
+      Spacing for the Small t-shirt size.
+    markup: |
+      <div class="spectrum-ButtonGroup--sizeS">
```

Comment:
line 26 should be 
`<div class="spectrum-ButtonGroup spectrum-ButtonGroup--sizeS">`

(missing spectrum-ButtonGroup base class)

Reply:
🙈 Oops, thank you!